package.json

```diff
@@ -22,7 +22,8 @@
     "*": "prettier --write --ignore-unknown",
     "packages/**/*.{ts,tsx,js,jsx}": "eslint -c packages/.eslintrc.json --fix",
     "packages/mask/**/*": "node ./packages/scripts/src/bin/precommit-hook-localekit.js",
-    "packages/web3-constants/**/*": "pnpm start --filter web3-constants"
+    "packages/web3-constants/**/*": "pnpm start --filter web3-constants",
+    "cspell.json": "sh -c 'gulp reorderSpellcheck'"
```

Comment:
```suggestion
    "cspell.json": "gulp reorderSpellcheck"
```

Reply:
https://github.com/okonet/lint-staged/issues?q=is%3Aissue+gulp+is%3Aclosed